src/components/common/ProgressBar/ProgressBar.jsx

```diff
@@ -1,25 +1,51 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-
+import { buildStyles, CircularProgressbarWithChildren } from 'react-circular-progressbar';
+import 'react-circular-progressbar/dist/styles.css';
+import { setProgressBarPathColor, setProgressBarTextColor } from '../../../utils/common/formatters/formmatters';
 import './_ProgressBar.scss';
 
 const ProgressBar = (props) => {
-  const { progress } = props;
+  const { progress, profileImg, onClickImage } = props;
+  const pathColorStyle = setProgressBarPathColor(progress);
+  const textColorStyle = setProgressBarTextColor(progress);
   return (
-    <div className="progress-bar">
-      <div className="progress" style={{ width: `${progress}%` }}>
-        <span>{progress}%</span>
-      </div>
+    <div className="progress-container">
```

Comment:
Use container component
```suggestion
    <Container className="progress-bar__container">
```

Reply:
done